docs/sources/configure-client/go-span-profiles.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Go span profiles with trace-to-profiles
+sidebar_label: Go span profiles
+description: Go span profiles with trace-to-profiles
+weight: 100
+---
+
+# Go span profiles with trace-to-profiles
+
+![span-profiles screenshot](https://grafana.com/static/img/docs/tempo/profiles/tempo-profiles-Span-link-profile-data-source.png)
+
+## Prerequisites
+
+1. Profiling: You will need to have your application instrumented with Pyroscopes Go SDK. If you haven't done this yet, please refer to the [Go (push mode)]({{< relref "../configure-client/language-sdks/go_push" >}}) guide.
```

Comment:
Just a note: strictly speaking, use of SDK is not required as tracing integration also works in pull mode. However, I do support the idea that we tell users to use SDK, if they want to integrate with tracing, because in this way it's just easier to control which Pyroscope labels match trace tags, e.g., using the same `service_name`, etc